opentracing/src/main/java/opentracing/TraceContext.java

```diff
@@ -1,66 +0,0 @@
-/**
- * Copyright 2016 The OpenTracing Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-package opentracing;
-
-/**
- * TraceContext encpasulates the smallest amount of state needed to describe a Span's identity
- * within a larger [potentially distributed] trace. The TraceContext is not intended to encode the
- * span's operation name, timing, or log data, but merely any unique identifiers (etc) needed to
- * contextualize it within a larger trace tree.
- *
- * <p>TraceContexts are sufficient to propagate the, well, *context* of a particular trace between
- * processes.
- *
- * <p>TraceContext also support a simple string map of "trace attributes". These trace attributes
- * are special in that they are propagated *in-band*, presumably alongside application data. See the
- * documentation for {@link #setAttribute(String, String)} for more details and some important
- * caveats.
- */
-public interface TraceContext {
```

Comment:
can't wait till we have a general DCP decoupled from Tracing, so can stop dancing around where we put things like this.